00-get-started/00-install-rw-kafka-pg.md

```diff
@@ -32,17 +38,6 @@ If you would like to explore other ways of installing RisingWave, see the [Quick
 
 Apache Kafka is an open-distributed event streaming platform for building event-driven architectures, enabling you to retrieve and process data in real time. 
 
-To install and run the self-hosted version of Kafka, follow steps one through three outlined in this [Apache Kafka quickstart](https://kafka.apache.org/quickstart).
-
-You should have learned how to install Kafka, start the environment, and create a topic. 
```

Comment:
Sounds too imperative, perhaps.

Reply:
Maybe we can provide more detailed commands here to help the reader to install the Kafka.

Reply:
Apart from installing Java in advance, all commands can be found in the link.